apps/safe-claiming-app/src/widgets/Snapshot.tsx

```diff
@@ -0,0 +1,188 @@
+import { Box, Chip, Link, Typography, Skeleton, styled } from "@mui/material"
+
+import useSafeSnapshot, {
+  type SnapshotProposal,
+} from "src/hooks/useSafeSnapshot"
+
+export const _getProposalNumber = (title: string): string => {
+  // Find anything that matches "SEP #n"
+  const SEP_REGEX = /SEP\s#\d+/g
+  return title.match(SEP_REGEX)?.[0] || ""
+}
+
+export const _getProposalTitle = (title: string): string => {
+  // Find anything after "] " or ": "
+  const TITLE_REGEX = /(\]|:) (.*)/
+  return title.match(TITLE_REGEX)?.at(-1) || ""
+}
+
+const SpaceContent = styled("div")`
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`
+
+const Proposal = styled("a")`
+  width: 100%;
+  display: grid;
+  grid-gap: 4px;
+  align-items: center;
+  text-decoration: none;
+  padding: 16px;
+  border: 1px solid #dcdee0;
+  border-radius: 6px;
+  grid-template-columns: 70px minmax(auto, 9fr) 1fr 1fr;
+  grid-template-areas: "number title title title title title title title title title status link";
+`
+
+const StyledChip = styled(Chip)`
+  grid-area: "status";
+  border-radius: 20px;
+  min-width: 68px;
+  text-align: center;
+  height: 23px;
+  color: #fff !important;
+  font-weight: bold;
+`
+
+const StyledNumber = styled(Box)`
+  grid-area: "number";
+  border-radius: 6px;
+  padding: 0px 8px;
+  color: black;
+  white-space: nowrap;
+`
+
+const StyledExternalLink = styled(Link)`
+  display: flex;
+  align-items: center;
+  font-weight: 700;
+  gap: 8px;
+  text-decoration: none;
+`
+
+const SnapshotProposals = ({
+  proposals,
+}: {
+  proposals: SnapshotProposal[]
+}) => (
+  <>
+    {proposals?.map((proposal) => (
+      <Proposal
+        href={`${SNAPSHOT_LINK}/proposal/${proposal.id}`}
+        key={proposal.id}
+        target="_blank"
+        rel="noopener noreferrer"
+        sx={{
+          "&:hover": {
+            borderColor: "primary.light",
+          },
+        }}
+      >
+        <StyledNumber
+          sx={{
+            backgroundColor: "#DCDEE0",
+          }}
+        >
+          {_getProposalNumber(proposal.title)}
+        </StyledNumber>
+        <Box gridArea="title">
+          <Typography
+            overflow="hidden"
+            color="text.primary"
+            sx={{
+              textOverflow: "ellipsis",
+              whiteSpace: "nowrap",
+            }}
+          >
+            {_getProposalTitle(proposal.title)}
+          </Typography>
+        </Box>
+        <StyledChip
+          label={proposal.state}
+          sx={{
+            backgroundColor:
+              proposal.state === "active" ? "success.main" : "#743EE4",
+          }}
+        />
+        <Box gridArea="link" display="flex" alignItems="center">
+          <OpenInNewRounded fontSize="small" sx={{ color: "text.primary" }} />
+        </Box>
+      </Proposal>
+    ))}
+  </>
+)
+
+const SNAPSHOT_LINK = "https://snapshot.org/#/safe.eth"
+const FORUM_LINK = "https://forum.gnosis-safe.io"
```

Comment:
Should we go with `https://forum.safe.global`?

Reply:
overlooked it! Totally

Reply:
There's a redirect `https://forum.safe.global` -> `https://forum.gnosis-safe.io/`
I guess it is preferable to leave as per your suggestion right?

Reply:
Regardless of redirection, I would use `safe.global` links where possible.